pipelines/vars/katello_devel.yml

```diff
@@ -13,4 +13,4 @@ foreman_installer_additional_packages:
 foreman_installer_options_internal_use_only:
   - "--disable-system-checks"
   - "--katello-devel-enable-ostree=true"
```

Comment:
You could drop this since it's the default

Reply:
But out of scope for this PR IMO

Reply:
True